src/player_display.cpp

```diff
@@ -342,7 +342,7 @@ static void draw_proficiencies_tab( const catacurses::window &win, const unsigne
         if( !cur.known && cur.id->can_learn() ) {
             static_assert( grid_width == 26, "Reminder to update formatting"
                            "for this string when grid width changes" );
-            name = string_format( "%-22s%2.0f%%", trim_by_length( cur.id->name(), width - 4 ),
+            name = string_format( "%-22s%2.0f%%", trim_by_length( cur.id->name(), width - 3 ),
                                   cur.practice * 100 );
```

Comment:
This should resolve Kelenius's issue
```suggestion
                                  std::floor( cur.practice * 100 ) );
```